components/brave_sync/brave_sync_service_impl.cc

```diff
@@ -463,6 +464,7 @@ void BraveSyncServiceImpl::OnResolvedPreferences(const RecordsList& records) {
   bool contains_only_one_device = false;
 
   auto sync_devices = sync_prefs_->GetSyncDevices();
+  bool waiting_for_second_device = sync_devices->size() < 2;
```

Comment:
can you add a method for this? There are two checks that both check the same condition, but with different values `<= 1` vs `< 2` and it's confusing to read like that